src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -226,7 +226,7 @@ public static double BitIncrement(double x)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static double CopySign(double x, double y)
         {
-            if (Sse.IsSupported || AdvSimd.IsSupported)
+            if (Sse2.IsSupported || AdvSimd.IsSupported)
             {
```

Comment:
Was this a typo?

Reply:
No. There is little need to differentiate on SSE vs SSE2 considering both are considered baseline to RyuJIT. This just simplifies the overall logic.